src/test/java/net/sf/jabref/exporter/ExportFormatTest.java

```diff
@@ -0,0 +1,69 @@
+package net.sf.jabref.exporter;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import com.google.common.base.Charsets;
+
+import net.sf.jabref.Globals;
+import net.sf.jabref.JabRefPreferences;
+import net.sf.jabref.MetaData;
+import net.sf.jabref.logic.journals.JournalAbbreviationLoader;
+import net.sf.jabref.model.database.BibDatabase;
+import net.sf.jabref.model.entry.BibEntry;
+
+public class ExportFormatTest {
+
+    @Before
+    public void setUp() {
+        Globals.prefs = JabRefPreferences.getInstance();
+        ExportFormats.initAllExports();
+        Globals.journalAbbreviationLoader = new JournalAbbreviationLoader(Globals.prefs);
+
+    }
+
+    @Test
+    public void testExportingEmptyDatabaseLayoutBasedFormat() throws Exception {
+        BibDatabase db = new BibDatabase();
```

Comment:
I probably copies that line from the exactly same code that was later
commented on. I'll fix it. :-)
Den 21 mar 2016 18:03 skrev "Tobias Diez" <notifications@github.com>:

> In src/test/java/net/sf/jabref/exporter/ExportFormatTest.java
> <https://github.com/JabRef/jabref/pull/1010#discussion_r56858544>:
>
> > +import net.sf.jabref.model.database.BibDatabase;
> > +import net.sf.jabref.model.entry.BibEntry;
> > +
> > +public class ExportFormatTest {
> > +
> > +    @Before
> > +    public void setUp() {
> > +        Globals.prefs = JabRefPreferences.getInstance();
> > +        ExportFormats.initAllExports();
> > +        Globals.journalAbbreviationLoader = new JournalAbbreviationLoader(Globals.prefs);
> > +
> > +    }
> > +
> > +    @Test
> > +    public void testExportingEmptyDatabaseLayoutBasedFormat() throws Exception {
> > +        BibDatabase db = new BibDatabase();
>
> Not sure how strict we are, but db is an abbreviation.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/JabRef/jabref/pull/1010/files/7e0bd7c2e6254337ce04085bd5eb3ebd86479c43#r56858544>
>